lib/ark_elixir/delegate.ex

```diff
@@ -33,7 +33,7 @@ defmodule Ark_Elixir.Delegate do
          "publicKey" => "038dfc041c7e609f254b2cf38de4b55e02dff9e743497f5cf6b67d49d8e44978ce",
          "username" => "drusilla", "vote" => "0"}], "success" => true}
     """
-    def search_delegates(query, opts \\ []) do
+    def search_delegates(query, _opts \\ []) do
```

Comment:
Small fix not mentioned in the commit message. The compiler was yelling about it not being used. I just modified it a little to make the compiler happy.